xml/security_ldap.xml

```diff
@@ -405,11 +409,80 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
 
   <sect2 xml:id="sec.security.ldap.server.setup">
   <title>Installation and Basic Setup</title>
-   <para>Install the 389-ds package:</para>
+   <para>
+    The <systemitem>389-ds</systemitem> package contains the &ds398; and the
+    administration tools. If the package is not installed yet, install it with
+    the following command:</para>
   <screen>&prompt.sudo;zypper install 389-ds</screen>
- <!--next steps:
-     create instance.inf
-     dscreate from-file etc.-->
+ <procedure>
+  <title>Setting up the &ds389; Instance</title>
+    <para>
+     You create the instance with the <command>dscreate</command> command.
+     It can take a configuration file (<filename>*.inf</filename>) which
+     defines the instance configuration settings. Alternatively, it can be run
+     in an interactive mode. To create a template <filename>*.inf</filename> file,
+     use the <option>create-template</option> option. For more details, see the
```

Comment:
Sounds good, you can probably add that the template is commented and pre-filled, so you can just read it and tweak variables. :)

Reply:
Done.